lib/cisco_node_utils/cmd_ref/overlay_global.yaml

```diff
@@ -3,7 +3,7 @@
 _exclude: [ios_xr]
 
 anycast_gateway_mac:
-  _exclude: [N3k]
+  _exclude: [N3k, N7k]
```

Comment:
Possibly a license or h/w requirement. Trust me, I ran the overlay test on my n7k and this property works.